### ISSUE TITLE
Proposal: triage security/issues/165

### ISSUE THREAD

**Issue by lucasgonze** [magma/magma#15371]
# Problem

The researcher Nathaniel Bennett submitted a package of 10 items. Ensuring timely fixes will require a substantial amount of time. 

# Solution

Fund a bounty to perform the following tasks:

- Review every item
- Create ten tickets
- Create an epic and possibly a kanban to manage the ten tickets
- Project manage the process of getting these fixed, including reproduction, testing, coding, CI/CD, and CVE management.
- Perform the engineering that makes sense for me personally

# Non-goals

Engineering on code that is out of scope for me.

# Acceptance

A vote will be taken in the TSC to consider the work has been completed.

On completion a payment of $4000 will be made.

**Comment by lucasgonze** [comment]
Votes so far, over in email:
Jordan: +1
Lucas: +1
Raphael: TBD
Pravin: TBD
Shubham: TBD
Yogesh: TBD

**Comment by lucasgonze** [comment]
Yogesh: +1
Raphael: +1

The motion is carried.

**Comment by lucasgonze** [comment]
1. Acceptance of all items except for project management of fixes is ready.
2. That remaining item must wait for 1.9.2.
3. Jordan proposes that I can invoice 80% before then, hold off on 20%.

**Comment by lucasgonze** [comment]
Acceptance vote:
+1 Yogesh

**Comment by lucasgonze** [comment]
Resources:
* ticket: https://github.com/magma/security/issues/165
* board: https://github.com/orgs/magma/projects/24/views/1

**Comment by lucasgonze** [comment]
Moving this to the Done category in the Proposal Tracker because the TSC has accepted and payment has been made.

**Comment by lucasgonze** [comment]
Reopening because I incorrectly interpreted the state of this ticket. It should be:

* Invoice for 80% of $4000 = $3,200
* Move to Bottlenecked because waiting on 1.9.2 to start
